client/security.coffee

```diff
@@ -31,10 +31,13 @@ claim_wiki = () ->
     .then (response) ->
       if response.ok
         response.json().then (json) ->
-          ownerName = json.ownerName
-          window.isClaimed = true
-          window.isOwner = true
-          update_footer ownerName, true
+          if wiki.lineup.bestTitle() is 'Login Required'
```

Comment:
Would this still work if there was a page titled "Login Required" in the site itself?

Reply:
There is nothing in the client other than having a 'Login Required' page to indicate that login is required to view. Maybe it could also look for a difference between the title and slug. But, will only happen on login, and only if the last page in the lineup has the title 'Login Required'.